File: Python/1013.py
def maior(a, b, c):
    mai = (a + b + abs(a - b)) / 2
    if mai == a:
        mai = a
    else:
        mai = b
    mai = (mai + c + abs(mai - c)) / 2
    if mai == c:
        mai = c
    return int(mai)
n1, n2, n3 = map(int, input().split())
print(maior(n1, n2, n3), "eh o maior")

File: Javascript/1013.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split(' ');

let A = parseInt(lines.shift());
let B = parseInt(lines.shift());
let C = parseInt(lines.shift());
let maior_ab = (A + B + Math.abs(A - B)) / 2;
let maior = (C + maior_ab + Math.abs(C - maior_ab)) / 2;
console.log(maior, "eh o maior");
